JS/Lesson35/onur/my-app/src/App.jsx

```diff
@@ -1,41 +1,66 @@
-import './App.css';
-import { useEffect, useState } from 'react';
-import { BD_SEARCH_BASE_URL } from './urls.js';
-import { AppRouter } from './AppRouter.jsx';
-import { Navbar } from './components/Navbar'
-
-export const App = () => {
-	// 1. Fetch recipes data from an API (Get request, API key, useState to store data, useEffect)
-	// 2. Display the data (RecipeList component -> use <ul> to display data)
-	// 3. Search recipe (A new component SearchBar with a <form> element, text input, and submit functionality)
-
-	const [recipes, setRecipes] = useState([]);
-	// If you want to keep both initial recipes and recipes from search  on the screen at the same time use code below:
-	// const [searchResult, setSearchResult] = useState([]);
-
-	// Runs once of component render (component render = "sayfanin yenilenmesi")
-	useEffect(() => {
-		fetch(BD_SEARCH_BASE_URL)
-			.then((response) => response.json())
-			.then((data) => setRecipes(data.meals));
-	}, []);
+import { Box, ThemeProvider, createTheme } from "@mui/material";
+import { Navbar } from "./components/Navbar";
+import { useEffect, useState } from "react";
+import { BD_SEARCH_BASE_URL } from "./urls.js";
+import { AppRouter } from "./AppRouter";
+import { useRecipesDispatch } from "./modules/recipes/RecipesProvider";
+import { RecipeActionType } from "./modules/recipes/models";
+import { UserProvider } from "./modules/user/UserProvider";
+import { FavoritesProvider } from "./modules/user/FavoritesProvider";
 
-	// If you want to keep both initial recipes and recipes from search  on the screen at the same time use code below:
-	// useEffect(() => {
-	//   setRecipes(prevState => {
-	//     // compare searchResult with prevState of recipes
-	//     // only add values from searchResult array that are not present in the prevState array (recipes)
-	//   });
+const App = () => {
 
-	// },[recipes, searchResult]);
+	const dispatch = useRecipesDispatch();
 
-	return (
-		<div className='container'>
+	const [mode, setMode] = useState("light");
 
-			<Navbar />
+	const theme = createTheme({
```

Comment:
It's common to create a separate file for application theme and keep the theme there, them you import it where it's needed. This way your code would be cleaner :)